src/components/UserActions/UserActions.ts

```diff
@@ -264,6 +266,16 @@ export class UserActions extends Component {
         this.element.appendChild(element);
     }
 
+    private renderEnablePrompt() {
+        const element = document.createElement('div');
+        element.classList.add('coveo-no-actions');
+        element.innerText = l(`${UserActions.ID}_enable_prompt`);
+        element.style.padding = '1.5em';
```

Comment:
Why not apply the padding in CSS on the 'coveo-no-actions' class?

Reply:
This padding is a reduction from the 3em padding already in the css class to allow the message to be displayed cleanly on two lines instead of three. I assumed the original padding value had some thought behind it so I didn't want to overwrite it everywhere.

Reply:
🤔 hm maybe a custom css class and some css then? It is my opinion here, but I'm not a huge fan of styling an element over the style attribute and prefer to avoid it when reasonable/possible.

Reply:
I agree that it isn't ideal. I just wanted to avoid essentially duplicating the class. On the other hand, the class name "no-actions" isn't super appropriate either... Ok I'll change it.